build/tools/ExtractRuleDescFromJson/RuleDescExtractor.cs

```diff
@@ -87,26 +88,61 @@ private void PrepareDestinationDirectory()
         }
     }
 
-    private static IEnumerable<Rule> LoadRules(string file)
+    private static IEnumerable<PluginRule> LoadRules(string file)
     {
+        var options = new JsonSerializerOptions { PropertyNameCaseInsensitive = true };
+
         var json = File.ReadAllText(file);
-        return JsonSerializer.Deserialize<Rule[]>(json)
-            ?? Array.Empty<Rule>();
+        return JsonSerializer.Deserialize<PluginRule[]>(json, options)
+            ?? Array.Empty<PluginRule>();
     }
 
-    private void ProcessRule(Rule rule)
+    private void ProcessRule(PluginRule pluginRule)
     {
         try
         {
-            var xml = EnsureHtmlIsXml(rule.Description);
-            SaveRuleFile(rule, xml);
+            var descAsxml = EnsureHtmlIsXml(pluginRule.Description);
+
+            var slvsRule = new RuleInfo(
+                pluginRule.Language ?? throw new ArgumentNullException("language"),
+                pluginRule.Key ?? throw new ArgumentNullException("key"),
+                descAsxml,
+                pluginRule.Name ?? throw new ArgumentNullException("name"),
+                ConvertPluginSeverity(pluginRule.DefaultSeverity),
+                ConvertPluginIssueType(pluginRule.Type),
+                Convert.ToBoolean(pluginRule.IsActiveByDefault),
+                pluginRule.Tags ?? Array.Empty<string>()
+                );
+
+            SaveRuleFile(slvsRule);
         }
         catch (Exception ex)
         {
-            Logger.LogError($"Error processing rule. Rule key: {rule.Key}, file: {context.RuleJsonFilePath}, {ex.Message}");
+            Logger.LogError($"Error processing rule. Rule key: {pluginRule.Key}, file: {context.RuleJsonFilePath}, {ex.Message}");
         }
     }
 
+    private static RuleIssueSeverity ConvertPluginSeverity(string? pluginSeverity)
+        => pluginSeverity switch
+        {
+            "MINOR" => RuleIssueSeverity.Minor,
+            "MAJOR" => RuleIssueSeverity.Major,
+            "INFO" => RuleIssueSeverity.Info,
+            "BLOCKER" => RuleIssueSeverity.Blocker,
+            "CRITICAL" => RuleIssueSeverity.Critical,
+            _ => throw new ArgumentException("Invalid enum value for pluginSeverity" + pluginSeverity, nameof(pluginSeverity)),
```

Comment:
I probably didn't need this method - the enum value names match, so I could probably have just used `Enum.Parse`.